src/RoboSaverVirtualModule.sol

```diff
@@ -186,16 +198,16 @@ contract RoboSaverVirtualModule {
         bytes memory joinPoolPayload =
             abi.encodeWithSelector(IVault.joinPool.selector, BPT_STEUR_EURE_POOL_ID, _card, _card, request);
 
-        // 3. batch approval and join into a multicall
+        /// @dev Batch approval and pool join into a multicall
         IMulticall.Call[] memory calls_ = new IMulticall.Call[](2);
         calls_[0] = IMulticall.Call(address(EURE), approvalPayload);
         calls_[1] = IMulticall.Call(address(BALANCER_VAULT), joinPoolPayload);
+        bytes memory multicallPayload = abi.encodeWithSelector(IMulticall.aggregate.selector, calls_);
 
-        bytes memory multiCallPayalod = abi.encodeWithSelector(IMulticall.aggregate.selector, calls_);
-
-        delayModule.execTransactionFromModule(MULTICALL3, 0, multiCallPayalod, 1);
+        /// @dev Queue the transaction into the delay module
+        delayModule.execTransactionFromModule(MULTICALL3, 0, multicallPayload, 1);
```

Comment:
thinking if here on the @dev comment worth highligthing that in call type will be actually a `delegatecall`, wdyt?

Reply:
f5a18b5